tiled/server/router.py

```diff
@@ -556,11 +556,25 @@ def post_metadata(
             specs=body.specs,
         )
     else:
-        raise HTTPException(status_code=404, detail="Failed to write in this node.")
+        raise HTTPException(status_code=404, detail="This path cannot accept metadata.")
 
     return json_or_msgpack(request, {"key": key})
 
 
+@router.delete("/node/delete/{path:path}")
```

Comment:
I think sticking with `/node/metadata/` is better.

The name is slightly confusing but we essentially treat that route as the canonical home for the data so I think we should carry that pattern on here.